extensions/.htmlnanorc

```diff
@@ -0,0 +1,3 @@
+{
+  minifySvg: false
```

Comment:
Is this necessary for this interface to work properly? I'd like to keep this minification on is possible

Reply:
Parcel bundler doesn't work for SVG (with Vue Directives).
To use SVG component, I had to use this configuration.

Reply:
Ref https://github.com/parcel-bundler/parcel/issues/1124